code/game/gamemodes/gang/dominator.dm

```diff
@@ -51,23 +51,22 @@
 
 	health -= damage
 
-	switch(health)
-		if(101 to INFINITY)
-			if(prob(damage*2))
-				sparks.set_up(5, 1, src)
-				sparks.start()
-		if(1 to 100)
+	if(health > (maxhealth/2))
```

Comment:
Nope, it looks fine to me. The icon doesn't start looking damaged until it is less than half health.